keyboards/mini_ten_key_plus/readme.md

```diff
@@ -0,0 +1,24 @@
+# mini_ten_key_plus
+
+![mini_ten_key_plus](imgur.com/QtFFONg.jpg)
+
+*A short description of the keyboard/project*
```

Comment:
fill this out or remove it. 
```suggestion
```

Reply:
Missed that one, I've since added a description and accepted the other two changes.